### PR TITLE
Add default log agent configuration file

### PR DESCRIPTION
**Pull request by haruki-yamanashi**
Only path of input section in log agent configuration could be set by
running installer with arguments "input_file_path_n". This change makes
it possible that the other following configuration could be set by
preparing default configuration file.

input section:
- add_field
- tags

filter section:
- filter name
- negate
- pattern
- what

### CODE REVIEW

**Comment by haruki-yamanashi** [comment]
@digitalfishpond @witekest

File: input.ini [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+add_field = { "dimensions" => { "service" => "neutron" "component" => "neutron-openvswitch-agent" }}
+tags = ["openstack"]
+path = "/var/log/neutron/openvswitch-agent.log"
+
+[neutron-dhcp-agent]
+add_field = { "dimensions" => { "service" => "neutron" "component" => "neutron-dhcp-agent" }}
+tags = ["openstack"]
+path = "/var/log/neutron/dhcp-agent.log"
+
+[neutron-ovs-clenup]
```
Comment (digitalfishpond) [inline input.ini]: neutron-ovs-cle**a**nup ?

File: input.ini [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+add_field = { "dimensions" => { "service" => "neutron" "component" => "neutron-ovs-clenup" }}
```
Comment (digitalfishpond) [inline input.ini]: as above

File: configure_log_agent.sh [hunk 3]
```diff
@@ -34,7 +34,7 @@ install_system_service() {
     sudo systemctl start monasca-log-agent
 }
 
-generate_config_file() {
+generate_specific_config_file() {
```
Comment (digitalfishpond) [inline configure_log_agent.sh:37]: line 56 curently checks if $FILES is empty, but that code will now never be reached.

File: configure_log_agent.sh [hunk 4]
```diff
@@ -163,8 +181,12 @@ echo PROJECT_DOMAIN_NAME  = "${PROJECT_DOMAIN_NAME}"
 echo DIMENSIONS    = "[ \"hostname:$HOSTNAME\"]"
 echo -e INPUT FILE\(S\) PATH\(S\) = "${FILES}"
 
-# Generate agent.conf file with specified values
-generate_config_file
+# Generate agent.conf file
+if [ ! "${FILES}" ]; then
```
Comment (digitalfishpond) [inline configure_log_agent.sh]: use `-z` switch (checks that string is empty) rather than `!` as $FILES does exist, it's just empty.

**Review by digitalfishpond — CHANGES_REQUESTED** [review]
Couple of thoughts to address.

File: configure_log_agent.sh (hunk 3 above)
Comment (haruki-yamanashi) [reply configure_log_agent.sh:37]: Done.

File: configure_log_agent.sh (hunk 4 above)
Comment (haruki-yamanashi) [reply configure_log_agent.sh]: Done.

File: input.ini (hunk 1 above)
Comment (haruki-yamanashi) [reply input.ini]: yes, cleanup is correct.
I fixed.

File: input.ini (hunk 2 above)
Comment (haruki-yamanashi) [reply input.ini]: yes, cleanup is correct.
I fixed.

**Review by digitalfishpond — APPROVED** [review]

**Pull request merged**